src/main/java/com/nsfl/gocrush/DBLayer/CrushRepository.java

```diff
@@ -1,7 +1,29 @@
-
 package com.nsfl.gocrush.DBLayer;
 
+import com.nsfl.gocrush.ModelLayer.Crush;
+import java.sql.*;
 
 public abstract class CrushRepository {
-    
+
+    protected Connection con;
+    protected Statement stat;
+    protected ResultSet rs;
+
+    public CrushRepository() {
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+            con = DriverManager.getConnection(
+                    "jdbc:mysql://localhost:3306/gocrush", "root", "NoraKhaled1932014");
```

Comment:
Create a Singleton class `AppConfig` that holds all configurations including `dbUrl`, `dbUsername`, `dbPassword`, ...etc